src/dataflow-types/src/client/controller/storage.rs

```diff
@@ -472,8 +511,13 @@ where
                 client
             });
 
+            let since = Antichain::from_elem(T::minimum());
```

Comment:
can we have a comment here, explaining why this only happens when we first create a source? (also, does it? if we restart environmentd wont this be rerun?)

Reply:
dont we need to read this from persist as well?